docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element if the user navigates back to the same
+page.
+
+### `{inputMapper.base}.thumbstickHistoryNavigation(that, value, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `invert {Boolean}` Whether the thumbstick history navigation should be in opposite order.
+- Returns: Nothing.
+
+Calls the [`{inputMapper.base}.previousPageInHistory`](#inputmapperbasepreviouspageinhistorythat-value) and
+[`{inputMapper.base}.nextPageInHistory`](#inputmapperbasenextpageinhistorythat-value) invokers according to the
+direction the `thumbstick` is pressed. The direction can be inverted by passing `true` as the `invert` argument.
+
+### `{inputMapper.base}.reverseTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for backward tab navigation, i.e. `reverseTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous tabbable element using gamepad `buttons` and
+`triggers`. The previous tabbable element is the element before the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.forwardTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for forward tab navigation, i.e. `forwardTab`.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the next tabbable element using gamepad `buttons` and
+`triggers`. The next tabbable element is the element next to the currently focused element in the array stored as
+`tabbableElements` member variable.
+
+### `{inputMapper.base}.thumbstickTabbing(that, value, speedFactor, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `speedFactor {Number}` Times by which the speed of tab navigation should be increased.
+- `invert {Boolean}` Whether the thumbstick tab navigation should be in opposite order.
+- Returns: Nothing.
+
+Changes the focus from the currently focused element to the previous or next tabbable element using gamepad
+`thumbsticks`. It calls the [`{inputMapper.base}.reverseTab`](#inputmapperbasereversetabthat-value-direction) and
+[`{inputMapper.base}.forwardTab`](#inputmapperbaseforwardtabthat-value-direction) invokers according to the direction
+the thumbstick is pressed and works only in **continuous** mode. The direction can be inverted by passing `true` as
+the `invert` argument, and the speed of tab navigation can be changed by changing the value of `speedFactor` argument.
```

Comment:
I see a lot of overlap between your descriptions and the variables/arguments.  I would suggest using the description to cover information about the function's behaviour in general, and specifics that are too long for the argument signature.   In any case you shouldn't be spelling out the same thing in two places so close to each other.